### ISSUE TITLE
Alias no longer works on Custom Commands

### ISSUE THREAD

**Issue by GostOmega** [Cog-Creators/Red-DiscordBot#342]
From the most recent commit on the alias cog, [#303](https://github.com/neonobjclash/Red-DiscordBot/commit/1c7f6ba492248aa6b9d2f4a3b4724669ca8ff061), I had issues to get the custom commands I've built to work that has been shortened with alias. I use this method to keeps things organized and simple for everyone on the server. I don't think the intention was to have alias no longer read custom commands

[Picture 1](https://puu.sh/qj5u3/fef844703b.png) shows the alias is responding to the server and working with a command `[p]ping` from _general.py_ under the alias `[p]test`. This also shows that alias can read my custom command `[p]commands` under the alias `[p]com`, yet it does not execute the command when I type in the alias.

[Picture 2](https://puu.sh/qj5yg/6bf870655a.png) shows that Custom Commands is working when I type in `[p]commands` and you can see the huge wall of text I set up for it. So I know the issue was not from _customcom.py_.

Basically, I use the alias to organize those giant wall text as sort of the server help page, but separated into categories. For example:
[p]com = [p]commands = General Commands and Reminder to read the Rules.
[p]com2 = [p]commands2 = Alias, CustomCom, and Fun Commands.
[p]slothelp = [p]economyhelp = List of Economy Commands and Reminder to register and play in a specific text channel.
Few more I won't list...

I reverted back to the previous version of the alias cog, waiting to see if this will work with custom commands again. Thank you for taking your time to read this!

**Comment by Kowlin** [comment]
As far as I know Aliases on CC's never worked, and this is never intended to work this way. If I recall correctly, there was a recent comment about this (https://github.com/Twentysix26/Red-DiscordBot/issues/327#issuecomment-230506391)

**Comment by GostOmega** [comment]
I see, so maybe we can approach this differently. I now know the alias is working as intended, and I see the reason why it shouldn't execute custom commands, thank you for reminding me on [#327](https://github.com/Twentysix26/Red-DiscordBot/issues/327#issuecomment-230506391).

Would it be possible to have Custom Commands having the ability to make it's own aliases, separate from the alias cog. This way, the alias cog can work on normal bot commands as intended, and the customcom can work with a new command like `[p]aliascom [new alias] [custom command]` for example, where this command can only read from `data\customcom\commands.json`. Maybe store the customcom aliases inside a aliascom.json file? What are your thoughts @Kowlin ?

I'm considering this idea to be a low priority for updates with the bot.

**Comment by Kowlin** [comment]
in theory its do-able, but in practise it would be easier for everyone if the custom commands are just copied across into another custom command of the users choice. This could be something for a 3rd party cog as I think that nobody from the team will see the value for it in the main bot.

**Comment by Kowlin** [comment]
Duplicate of the closed #327, closing
